app/src/main/java/com/example/foodjuju/ui/DAO/FoodJuJuDAO.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.foodjuju.ui.DAO
+
```

Comment:
An empty class is not adding much value...  it's clutter.  Only completed features should be merged to master.